locale/ja/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: index.hbs
 labels:
-  banner: Important security releases, please update now!
+  banner: Spectre and Meltdown in the context of Node.js - no current action required.
```

Comment:
The OS needs to be updated. That's more than "no current action required".

Reply:
I was on the edge on that, I'll come up with something better.

Reply:
Perhaps, "No Node.js specific action required"?

Reply:
Actually probably better to have people just read the blog so will remove the '- no current action required.'